network/message_authorization_config.go

```diff
@@ -0,0 +1,300 @@
+		},
+	}
+	blockVote = MsgAuthConfig{
+		String: "BlockVote",
+		config: map[Channel]flow.RoleList{
+			ConsensusCommittee: {flow.RoleConsensus},
+		},
+	}
+
+	// protocol state sync
+	syncRequest = MsgAuthConfig{
+		String: "SyncRequest",
+		config: map[Channel]flow.RoleList{
+			SyncCommittee:     flow.Roles(),
+			SyncClusterPrefix: flow.Roles(),
+		},
+	}
+	syncResponse = MsgAuthConfig{
+		String: "SyncResponse",
+		config: map[Channel]flow.RoleList{
+			SyncCommittee:     flow.Roles(),
+			SyncClusterPrefix: flow.Roles(),
+		},
+	}
+	rangeRequest = MsgAuthConfig{
+		String: "RangeRequest",
+		config: map[Channel]flow.RoleList{
+			SyncCommittee:     flow.Roles(),
+			SyncClusterPrefix: flow.Roles(),
+		},
+	}
+	batchRequest = MsgAuthConfig{
+		String: "BatchRequest",
+		config: map[Channel]flow.RoleList{
+			SyncCommittee:     flow.Roles(),
+			SyncClusterPrefix: flow.Roles(),
+		},
+	}
+	blockResponse = MsgAuthConfig{
+		String: "BlockResponse",
+		config: map[Channel]flow.RoleList{
+			SyncCommittee:     flow.Roles(),
+			SyncClusterPrefix: flow.Roles(),
+		},
+	}
+
+	// cluster consensus
+	clusterBlockProposal = MsgAuthConfig{
+		String: "ClusterBlockProposal",
+		config: map[Channel]flow.RoleList{
+			ConsensusClusterPrefix: {flow.RoleCollection},
+		},
+	}
+	clusterBlockVote = MsgAuthConfig{
+		String: "ClusterBlockVote",
+		config: map[Channel]flow.RoleList{
+			ConsensusClusterPrefix: {flow.RoleCollection},
+		},
+	}
+	clusterBlockResponse = MsgAuthConfig{
+		String: "ClusterBlockResponse",
+		config: map[Channel]flow.RoleList{
+			ConsensusClusterPrefix: {flow.RoleCollection},
+		},
+	}
+
+	// collections, guarantees & transactions
+	collectionGuarantee = MsgAuthConfig{
+		String: "CollectionGuarantee",
+		config: map[Channel]flow.RoleList{
+			PushGuarantees: {flow.RoleCollection}, // channel alias ReceiveGuarantees = PushGuarantees
+		},
+	}
+	transaction = MsgAuthConfig{
+		String: "Transaction",
+		config: map[Channel]flow.RoleList{
+			PushTransactions: {flow.RoleCollection}, // channel alias ReceiveTransactions = PushTransactions
+		},
+	}
+	transactionBody = MsgAuthConfig{
+		String: "TransactionBody",
+		config: map[Channel]flow.RoleList{
+			PushTransactions: {flow.RoleCollection}, // channel alias ReceiveTransactions = PushTransactions
+		},
+	}
+
+	// core messages for execution & verification
+	executionReceipt = MsgAuthConfig{
+		String: "ExecutionReceipt",
+		config: map[Channel]flow.RoleList{
+			PushReceipts: {flow.RoleExecution}, // channel alias ReceiveReceipts = PushReceipts
+		},
+	}
+	resultApproval = MsgAuthConfig{
+		String: "ResultApproval",
+		config: map[Channel]flow.RoleList{
+			PushApprovals: {flow.RoleVerification}, // channel alias ReceiveApprovals = PushApprovals
+		},
+	}
+
+	// execution state synchronization
+	executionStateSyncRequest = MsgAuthConfig{
+		String: "",
+		config: nil,
+	}
+	executionStateDelta = MsgAuthConfig{
+		String: "",
+		config: nil,
+	}
```

Comment:
that's this? If these are supposed to be undefined, can you add a comment explaining

Reply:
Added [deprecated] to the comment https://github.com/onflow/flow-go/pull/2491/commits/d40fc606ec58e98592cc2fbf5553ed5b47bb0ae1